recipes/compas_rrc/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "compas_rrc" %}
+{% set version = "1.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/compas_rrc-{{ version }}.tar.gz
+  sha256: 3078fb040d1e8a6ac776fcffc4edda63b0586acaff379214831ec5bc7c7f8577
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - compas_fab
+    - roslibpy >=1.1
+    - python >=3.6
+
+test:
+  imports:
+    - compas_rrc
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/compas-rrc/compas_rrc
+  summary: 'COMPAS RRC: Online control for ABB robots over a simple-to-use Python interface.'
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - gonzalocasas
+    - fleischp
```

Comment:
@fleischp are you OK becoming a maintainer here?

Reply:
that is fine for me.